build_tools/release_to_github.py

```diff
@@ -162,18 +182,22 @@ def is_main_file(path):
         filename = re.sub(r'https://%s/archive/(.*?)/' % config.archive_path, '', download_url)
         basename = os.path.basename(filename)
         # file stream upload to GitHub
-        log("Uploading to GitHub " + filename)
         with open(filepath, 'rb') as f:
             content_type,_ = mimetypes.guess_type(basename)
             headers = { "Content-Type": content_type or "application/octet-stream" }
             name = filename
-            if is_main_file(path):
+            if is_main_file(download_url):
```

Comment:
Python keeps references to old locals, and the "path" wasn't part of this for loop :(